pysatNASA/instruments/icon_mighti.py

```diff
@@ -67,66 +74,20 @@
-               for jj in sat_ids.keys()}
-_test_download_travis = {jj: {kk: False for kk in sat_ids[jj]}
-                         for jj in sat_ids.keys()}
-pandas_format = False
+inst_ids = {'': ['vector_wind_green', 'vector_wind_red'],
+            'a': ['los_wind_green', 'los_wind_red', 'temperature'],
+            'b': ['los_wind_green', 'los_wind_red', 'temperature']}
 
-datestr = '{year:04d}-{month:02d}-{day:02d}_v{version:02d}r{revision:03d}'
-fname1 = 'ICON_L2-1_MIGHTI-{id:s}_LOS-Wind-{color:s}_{date:s}.NC'
-fname2 = 'ICON_L2-2_MIGHTI_Vector-Wind-{color:s}_{date:s}.NC'
-fname3 = 'ICON_L2-3_MIGHTI-{id:s}_Temperature_{date:s}.NC'
-supported_tags = {'': {'vector_wind_green': fname2.format(color='Green',
-                                                          date=datestr),
-                       'vector_wind_red': fname2.format(color='Red',
-                                                        date=datestr)},
-                  'a': {'los_wind_green': fname1.format(id='A', color='Green',
-                                                        date=datestr),
-                        'los_wind_red': fname1.format(id='A', color='Red',
-                                                      date=datestr),
-                        'temperature': fname3.format(id='A', date=datestr)},
-                  'b': {'los_wind_green': fname1.format(id='B', color='Green',
-                                                        date=datestr),
-                        'los_wind_red': fname1.format(id='B', color='Red',
-                                                      date=datestr),
-                        'temperature': fname3.format(id='B', date=datestr)}}
-
-# use the CDAWeb methods list files routine
-list_files = functools.partial(mm_gen.list_files,
-                               supported_tags=supported_tags)
-
-# support download routine
-dirstr = '/pub/LEVEL.2/MIGHTI{id:s}'
-dirdatestr = '{year:4d}/{doy:03d}/'
-ids = {'': '',
-       'a': '-A',
-       'b': '-B'}
-products = {'vector_wind_green': 'Vector-Winds/',
-            'vector_wind_red': 'Vector-Winds/',
-            'los_wind_green': 'LOS-Winds/',
-            'los_wind_red': 'LOS-Winds/',
-            'temperature': 'Temperature/'}
-datestr = '{year:04d}-{month:02d}-{day:02d}'
-
-download_tags = {}
-for skey in supported_tags.keys():
-    download_tags[skey] = {}
-    for tkey in supported_tags[skey].keys():
-        fname = supported_tags[skey][tkey]
+pandas_format = False
 
-        download_tags[skey][tkey] = {'dir': dirstr.format(id=ids[skey]),
-                                     'remote_fname': ''.join((products[tkey],
-                                                              fname))}
+# ----------------------------------------------------------------------------
+# Instrument test attributes
 
-download = functools.partial(mm_icon.ssl_download, supported_tags=download_tags)
+_test_dates = {jj: {kk: dt.datetime(2020, 1, 2) for kk in inst_ids[jj]}
```

Comment:
```suggestion
_test_dates = {jj: {kk: dt.datetime(2020, 1, 2) for kk in inst_ids[jj].keys()}
```

Reply:
inst_ids[jj] is a list, and does not have keys